mesonbuild/compilers/c.py

```diff
@@ -554,6 +555,36 @@ def get_include_args(self, path, is_system):
             path = '.'
         return ['-I' + path]
 
+class CompCertCCompiler(CompCertCompiler, CCompiler):
+    def __init__(self, exelist, version, for_machine: MachineChoice,
+                 is_cross, info: 'MachineInfo', exe_wrapper=None, **kwargs):
+        CCompiler.__init__(self, exelist, version, for_machine, is_cross,
+                           info, exe_wrapper, **kwargs)
+        CompCertCompiler.__init__(self)
+
+    def get_options(self):
+        opts = CCompiler.get_options(self)
+        opts.update({'c_std': coredata.UserComboOption('C language standard to use',
+                                                       ['none', 'c89', 'c99'],
+                                                       'none')})
+        return opts
+
+    def get_option_compile_args(self, options):
+        return []
+
+    def get_no_optimization_args(self):
+        return ['-O0']
+
+    def get_output_args(self, target):
+        return ['-o%s' % target]
```

Comment:
```suggestion
        return ['-o{}'.format(target)]
```

I think we're trying to migrate away from using the % formatters.